src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -320,8 +321,19 @@ public void init(boolean forceNewConnection) {
         clearInstances(instances);
         clearInstances(brokenInstances);
 
+
         Reporter reporter = appConfig.getReporter();
 
+        String action = appConfig.getAction();
+
+        if( action.equals( AppConfig.ACTION_LIST_JVMS )) {
+            List<VirtualMachineDescriptor> descriptors = com.sun.tools.attach.VirtualMachine.list();
```

Comment:
Can we import `com.sun.tools.attach.VirtualMachine` along with `com.sun.tools.attach.VirtualMachineDescriptor` in the header ?